### ISSUE TITLE
使用GPU训练时会出现loss显示nan问题

### ISSUE THREAD

**Issue by Answer0327** [QyTnT/MaskRCNN#1]
一开始因为cuda少文件只能使用cpu训练，显示一切正常，当补全cuda文件后使用gpu训练时会出现nan问题，GPU是3090显存24G，例如：
Epoch 00012: LearningRateScheduler reducing learning rate to 2.9501769101577432e-05.
Epoch 12/105
48/48 [==============================] - 17s 358ms/step - loss: nan - rpn_class_loss_loss: nan - rpn_bbox_loss_loss: nan - mrcnn_class_loss_loss: 0.6799 - mrcnn_bbox_loss_loss: 0.0000e+00 - mrcnn_mask_loss_loss: 0.0000e+00 - val_loss: nan - val_rpn_class_loss_loss: nan - val_rpn_bbox_loss_loss: nan - val_mrcnn_class_loss_loss: 0.6792 - val_mrcnn_bbox_loss_loss: 0.0000e+00 - val_mrcnn_mask_loss_loss: 0.0000e+00 - lr: 2.9502e-05
其中很多损失显示都是nan，请问这个要如何解决，
